lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -19,7 +31,11 @@ def define
           if response == 'yes'
             question :working_tax_credit?
           else
-            outcome :outcome_not_affected_no_housing_benefit
+            if chosen_cap == 'future'
```

Comment:
See comment: https://github.com/alphagov/smart-answers/pull/2481#issuecomment-220296815 for why default and future were chosen as names.